### PR TITLE
feat: Added disclaimer to container-app ptn - `avm/ptn/azd/container-apps-stack`

### LINKED ISSUE #3219: [Failed pipeline] avm.ptn.azd.container-apps
Failed run: https://github.com/Azure/bicep-registry-modules/actions/runs/10741105381

### PR DESCRIPTION
**Pull request by AlexanderSehr**
## Description

- Added disclaimer to container-app ptn
- Regenerated files

## Pipeline Reference

| Pipeline |
| -------- |
[avm.ptn.azd.container-apps-stack](https://github.com/Azure/bicep-registry-modules/actions/workflows/avm.ptn.azd.container-apps-stack.yml)

## Type of Change

- [ ] Update to CI Environment or utilities (Non-module affecting changes)
- [x] Azure Verified Module updates:
  - [ ] Bugfix containing backwards-compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

### CODE REVIEW

File: avm/ptn/azd/container-apps/README.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 Creates an Azure Container Registry and an Azure Container Apps environment.
 
+**Note:** This module is not intended for broad, generic use, as it was designed to cater for the requirements of the AZD CLI product. Feature requests and bug fix requests are welcome if they support the development of the AZD CLI but may not be incorporated if they aim to make this module more generic than what it needs to be for its primary use case
```
Comment (AlexanderSehr) [inline avm/ptn/azd/container-apps/README.md:5]: May depend on https://github.com/Azure/bicep-registry-modules/pull/3234#discussion_r1755471847

**Comment by ReneHezser** [comment]
@AlexanderSehr 
We have added examples for Bicep parameter files to the Readme. This has been applied to all published modules but needs to be done for PRs as well. Can you please run the Set-AVMModule utility as detailed [here](https://azure.github.io/Azure-Verified-Modules/contributing/bicep/bicep-contribution-flow/generate-bicep-module-files/). It is required for the validation pipeline to succeed and the contribution to be published.

Please reach out if any support is needed.

**Comment by matebarabas** [comment]
As [agreed here](https://github.com/Azure/bicep-registry-modules/pull/3234#discussion_r1790522023), this module has been renamed to `avm/ptn/azd/container-apps-stack` (formerly, it was `avm/ptn/azd/container-apps`).

**Comment by AlexanderSehr** [comment]
> As [agreed here](https://github.com/Azure/bicep-registry-modules/pull/3234#discussion_r1790522023), this module has been renamed to `avm/ptn/azd/container-apps-stack` (formerly, it was `avm/ptn/azd/container-apps`).

updated

**Review by matebarabas — APPROVED** [review]
Looks good, thanks for incorporating the disclaimer!

**Comment by AlexanderSehr** [comment]
Closing in favor of new #3452

**Pull request closed without merging**